code/game/machinery/computer/shuttle.dm

```diff
@@ -61,6 +61,8 @@
 
 /obj/machinery/computer/shuttle/emag(mob/user as mob)
 	if(!emagged)

```

Comment:
Why wouldn't you just keep the machine flag? emagged is a deprecated legacy variable to that. And they work in the exact same way.